ci/jobs/build_clickhouse.py

```diff
@@ -155,7 +161,9 @@ def main():
         res = results[-1].is_ok()
 
     if res and JobStages.BUILD in stages:
-        Shell.check("sccache --show-stats")
+        run_shell("Env vars", "env | sort")
+        run_shell("sccache stats", "env | sccache --show-stats")
```

Comment:
How it differs from?

```suggestion
        run_shell("sccache stats", "sccache --show-stats")
```

Reply:
Leftoever from when I was doing everything in a single call :sweat_smile: